lib/workload/stateful/stacks/token-service/deploy/stack.ts

```diff
@@ -35,7 +36,7 @@ export class TokenServiceStack extends Stack {
     super(scope, id, props);
     this.props = props;
 
-    this.vpc = Vpc.fromLookup(this, 'MainVpc', props.vpcProps);
+    this.vpc = getVpc(this);
```

Comment:
Ah. You want the function one. Please check with the Flo! 😃 Think, he prefer the other way, `vpcProps` pass down. Leave it with you two.

Reply:
Hm,... does the function add much? 
It's a bit easier for me without... first I don't need to know about any outside function and second it's just a lookup like I'd need to do for any other (provided) resource.
Also, `getVpc()` hides away configuration details that otherwise would be in the usual, expected spot (config constants), right?

Reply:
Yup. Since we change to independent "stack" concept, the props password become more make sense.

Think, Will agree (somewhere ^ there comment; deprecating this function lookup all together). Perhaps, Will could help make necessary changes and, use `vpcProps` pass down. I reckon, that should work... But, leaving the final call to Will.

Reply:
Will remove it! It will be easier for native lookup rather than understanding a custom function.

Reply:
Yay! 🎉